StantUiIosLibrary/Classes/Tables/ConstructionTableViewWithCollapsedSearchBar/ConstructionTableViewWithCollapsedSearchBar.swift

```diff
@@ -79,16 +74,26 @@ public class ConstructionTableViewWithCollapsedSearchBar: UIView {
                                           animationDelegate: self,
                                           selectCellDelegate: tableViewDelegate)
         tableView?.backgroundColor = UIColor.clear
+        guard let tableView = tableView else { return }
+        self.addSubview(tableView)
```

Comment:
If here you just using the guard to add de tableView as subview, you could probably do this instead (saving 1 line, haha), would probably have the same effect:
```suggestion
        if let tableView = tableView { self.addSubview(tableView) }
```

Reply:
Sure :)